filters/f_hwtransfer.c

```diff
@@ -309,7 +309,7 @@ static bool probe_formats(struct mp_hwupload *u, int hw_imgfmt)
     }
 
     if (!ctx) {
-        MP_ERR(u->f, "no support for this hw format\n");
+        MP_INFO(u->f, "no support for this hw format\n");
```

Comment:
Can you explain why we want to do this (and below)? With other VOs, it will fail the upload and then fallback to trying to pass on the original frame. It seems undesirable to do that with no feedback.

Reply:
It's related to the change in `f_autoconvert.c`. Since that loops through all the formats now, it's possible for a fmt to fail, but that doesn't necessarily mean it's a hard error (the next format may succeed for instance). In the event that all of them fail, we could probably instead print the error in `build_image_converter` since it appears to be the only function that actually ends up calling this.